cmd/data.go

```diff
@@ -84,6 +85,7 @@ func (cmd *DataCmd) SetFlags(f *flag.FlagSet) {
 	f.StringVar(&cmd.target, "target", "Spanner", "Specifies the target DB, defaults to Spanner (accepted values: `Spanner`)")
 	f.StringVar(&cmd.targetProfile, "target-profile", "", "Flag for specifying connection profile for target database e.g., \"dialect=postgresql\"")
 	f.StringVar(&cmd.filePrefix, "prefix", "", "File prefix for generated files")
+	f.StringVar(&cmd.project, "project", "", "Flag spcifying default project id for all the generated resources for the migration")
```

Comment:
Since target profile also has a field named project and we are adding one here with the note: "Flag spcifying default project id for all the generated resources for the migration" it looks a bit confusing to me. Maybe we should rename to where it is used

Reply:
project field in target profile refers to Spanner's project id (since it is present in target profile). Project flag outside refers to migration project id which will be used to create other migration resources like Dataflow jobs, Datastream, Pubsub etc. Do you think anything else need to be mentioned in the flag description?